### ISSUE TITLE
Проверка лабораторной работы 2

### ISSUE THREAD

**Issue by Nevarkir** [Nevarkir/Second-lab#1]
To: @evgeny-p

**Comment by evgeny-p** [comment]
Замечания по работе с гитом:

1. Вся работа выполнена в ветке `master`, либо была выполнена в отдельной ветке и смержена без опции `--no-ff`. Чтобы исправить, нужно:
  1. Переключиться на ветку `master`: `git checkout master`.
  1. Создать на ее месте новую ветку: `git branch feature`.
  1. Откатить мастер на первый коммит: `git reset --hard <хэш первого коммита>`.
  1. Слить ветки: `git merge --no-ff feature`.
  1. Залить ветку `master` с опцией `--force`: `git push --force origin master`.
1. Артефакты сборки не следует коммитить. Нужно удалить `main.exe`.
1. Не нужно в заголовке коммита указывать его номер. Несколько коммитов могут быть объединены в один, один коммит может быть разделен на несколько, порядок коммитов может быть изменен. При этом актуальность таких заголовков теряется. Также ряд сложностей связан с нумерацией объектов в распределенных системах.

Замечания по коду:

1. Приложение компилируется с предупреждениями (gcc 4.8):
```
$ gcc -Wall lab.c -lm
lab.c:3:1: warning: return type defaults to ‘int’ [-Wreturn-type]
 main ()
 ^
lab.c: In function ‘main’:
lab.c:23:3: warning: implicit declaration of function ‘sqrt’ [-Wimplicit-function-declaration]
   x1 = (-b + sqrt(d)) / (2 * a);
   ^
lab.c:23:14: warning: incompatible implicit declaration of built-in function ‘sqrt’ [enabled by default]
   x1 = (-b + sqrt(d)) / (2 * a);
``` 
1. Опечатка в тексте: `s/Input/Input/`.

**Comment by Nevarkir** [comment]
Да, смержил без но оф, хорошо буду делать без нумерации. В тот день когда я скинул работу у меня сломался ноутбук, поэтому я пока не могу работать дома, кажется к понедельнику починят
